met-web/src/components/layout/Header/index.tsx

```diff
@@ -1,66 +1,74 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import AppBar from '@mui/material/AppBar';
 import Toolbar from '@mui/material/Toolbar';
 import { useAppSelector } from '../../../hooks';
+import { IconButton, Button, useMediaQuery } from '@mui/material';
 import UserService from '../../../services/UserServices';
-import {
-    buttonClass,
-    LogoContainer,
-    LogoutContainer,
-    TitleContainer,
-    sxLight,
-    AuthButton,
-    HeaderText,
-} from './HeaderElements';
+import { LogoContainer, LogoutContainer, TitleContainer, AuthButton, HeaderText } from './HeaderElements';
 import sx from 'mui-sx';
+import SideNav from '../SideNav/SideNav';
 
 const Header = () => {
     //states
     const isLoggedIn = useAppSelector((state) => state.user.authentication.authenticated);
+    const [open, setOpen] = useState(false);
+    const [width, setWidth] = useState(window.innerWidth);
```

Comment:
do we still need this?

Reply:
Open is needed for smaller screens width has been removed

Reply:
of course, I was referring to line 15 the width, setWidth